.gitignore

```diff
@@ -1,29 +1,124 @@
-HELP.md
-/target/
-!.mvn/wrapper/maven-wrapper.jar
-
-### STS ###
-.apt_generated
-.classpath
-.factorypath
-.project
-.settings
-.springBeans
-.sts4-cache
-
-### IntelliJ IDEA ###
-.idea
```

Comment:
Leave this in you don't want the .idea folder to be pushed to git. It's for intelij Ide